### ISSUE TITLE
Unable to access Simple EOS wallet

### ISSUE THREAD

**Issue by Terrytibbs1** [eosrio/simpleos#108]
Hello all.

I have a problem that hopefully someone here may be able to assist with.

I have a Simple Eos wallet that I HODl'd my Eos tokens for a couple of years. My laptop recently had some issues and the hard drive was replaced. 

I now am unable to access the wallet as I try to start it up I get a troubleshoot message saying its 'Couldn't connect to the blockchain, check your internet connection and try again' and another comment about try connecting to another end point.

The computer's connected to the internet and I have tried downloading updated versions of the wallet but they require Provate/public key details.

I've looked everywhere for my private and public keys but have been unable to find them. 

Basically is there any chance of me recovering this wallet account? I know the password, the pin number and the wallet address.

Thanks in advance, am happy to provide more info if required.

**Comment by seveballesteros** [comment]
I had the same issue.  Have you made sure you are on the correct network. Mine dropped out. Guessing you've probably got back in by now anyway?

**Comment by Terrytibbs1** [comment]
I have been unable to gain access to the tokens to this day. Could you please explain how you managed to get on the correct network?

**Comment by SatMan2000** [comment]
what wallet are you using?  may be easier to talk somewhere else.  i have several platforms. let me know which one suits you best and we can talk there.

**Comment by SatMan2000** [comment]
guess you sorted this?

**Comment by Terrytibbs1** [comment]
> what wallet are you using? may be easier to talk somewhere else. i have several platforms. let me know which one suits you best and we can talk there.

Hi there. Sorry about the slow response, I have been unable to resolve this matter yet. I was using the Simple EOS web wallet and had the issue following having my hard drive replaced by Apple following a performance issue. I would be interested in discussing this further. Which platform suits you best? 

Thanks

**Comment by Ilovmygsd** [comment]
I have the same problem.  I replaced my Mac a while back.  I don't know how long I haven't been able to get in because I made the wallet in 18..and have just let it sit.

**Comment by Ilovmygsd** [comment]
Ok Terry. I figured it out.  I googled simple eos wallet.  I went to their website and downloaded the newest version.  I had my private keys and password and I got in.

**Comment by SatMan2000** [comment]
hey terry.  hopefully llov has sorted this but not sure it will work if you don't have your keys?  message me on telegram if you are struggling and i'll try and help
@gdog33
